packages/shared/src/graphql/sources.ts

```diff
@@ -46,6 +46,7 @@ export interface Squad extends Source {
   members?: Connection<SourceMember>;
   membersCount: number;
   description: string;
+  memberPostingRank: number;
```

Comment:
Should this not be `memberPostingRole`?

Reply:
Yeah probably we can do that to uniform the edit function, but let's see once we tackle that one.